src/plugins/mf_priority.cpp

```diff
@@ -1007,7 +1063,9 @@ extern "C" int flux_plugin_init (flux_plugin_t *p)
     if (flux_plugin_register (p, "mf_priority", tab) < 0
         || flux_jobtap_service_register (p, "rec_update", rec_update_cb, p)
         || flux_jobtap_service_register (p, "reprioritize", reprior_cb, p)
-        || flux_jobtap_service_register (p, "rec_q_update", rec_q_cb, p) < 0)
+        || flux_jobtap_service_register (p, "rec_q_update", rec_q_cb, p)
+        || flux_jobtap_service_register (p, "rec_proj_update", rec_proj_cb, p)
+        < 0)
```

Comment:
Why the `< 0` check only for the last one in this `||` chain?

Reply:
Oops, I guess I forgot to include a `< 0` check for the other calls here. Thanks for catching this! I added another commit to this PR that adds checks for all of the `flux_jobtap_service_register ()` calls here